src/routes/api/chat.routes.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import path from 'path';
+
+const route = express.Router();
+
+route.get('/:id', (req, res) => {
+  const filePath = path.join(__dirname, '../../public', 'index.html');
+  res.sendFile(filePath);
+});
+
+route.use((err, req, res, next) => {
+  console.error(err.stack);
+  res.status(500).send('Something broke!');
+});
```

Comment:
1. Why this Please?
2. Also, let's have the same response format.
3. it should be in JSON format

Reply:
@Abdulkeza Fixed